mod/spm_errno.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <common.h>
+
+extern "C" {
```

Comment:
This should have a CPP_WRAPPER with the msl namespace. It'd also be ideal to have the official name of `errno` rather thatn `spm_errno`, but I don't know if that would affect compatibility with the network mod's setup at all

Reply:
annoyingly enough it does, I think ill just remove it from spm headers and put it in the include folder for now

Reply:
The symbol is still in the lsts

Reply:
found a way to keep errno.h in the msl namespace while also keeping compatibility with the network mod
__errno doesnt appear to be a real spm function, its just one that john made for this mod specifically, so I removed that from spm headers and made a .h file in the network mod's include so that the mod would still work

Reply:
But the symbol name is still spm_errno

Reply:
And this definitely doesn't break the network mod in any way?

Reply:
nope, I just had to change some of the local code in my fork
Ill probably make a PR to johns repo with the changed local code